### ISSUE TITLE
Design and implement question prompt

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#1126]
### Problem

We do not have a way of forcing the user to answer an important question.

Our modal component is dismissable, and what's needed is a centered rectangle that asks a question and puts up a grey overlay over the background so that it's unclickable and to focus the prompt question.

### Solution

Design this component. /cc @leihla 

Implement it such that it can be displayed immediately in the app using a single line, e.g.:

```js
sbp('gi.ui/prompt', L('my question'), { okButton: L('Yes'), cancelButton: L('No') })
```

EDIT: this has now been designed here: https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0?node-id=21598%3A194029

**Comment by leihla** [comment]
Looking at this scenario, and reading #1125, I've designed the question prompted assuming that we want to force the user to re-enter their income details after a change to the mincome has happened. However the undismissable modal can be used for other scenarios.

The modal is similar to other modals, without the close button, and should not be dismissed when clicking outside the modal area.

Link for figma: https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0?node-id=21526%3A193805

<img width="1441" alt="Screen Shot 2022-01-10 at 15 59 05" src="https://user-images.githubusercontent.com/158178/148797168-22de3289-3694-41bc-8dff-329954b99c5c.png">

**Comment by taoeffect** [comment]
@pieer As Leihla has designed this now: https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0?node-id=21598%3A194029

I'm assigning the issue to you to implement.

**Comment by taoeffect** [comment]
Closed by #1168
